nodepool/elements/jenkins-slave/install.d/20-jenkins-slave

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+##
+## Add jenkins user, group.
+##
+JENKINS_HOME="/var/lib/jenkins"
+groupadd jenkins
+useradd --gid jenkins \
+        --shell /bin/bash \
+        --home-dir ${JENKINS_HOME} \
+        --create-home jenkins
+
+##
+## Configure SSH
+##
+# this was copied from outside the chroot by extras.d
+_pub_key=/tmp/in_target.d/ssh-public-keys
+if [ ! -f $_pub_key ]; then
+    die "Can not find public keys!"
```

Comment:
Is the `die` function defined/imported anywhere? Unlink python, I think in bash you need to define it each time or in a common script??

Reply:
The ``die`` function is provided by diskimage-builder. See https://github.com/openstack/diskimage-builder/blob/958ea8a3372157fe62c3a2bb1fcd55a9cadedab1/diskimage_builder/lib/die#L19

Reply:
👍 cool, thanks for the link.